app/views/transactions/index.njk

```diff
@@ -41,11 +41,23 @@
 <div class="govuk-grid-column-full">
   <h1 class="govuk-heading-l">
     {% if allServiceTransactions %}
-       Transactions for all live services
+       Transactions for all services
+       <strong class="govuk-tag govuk-tag--grey">{{ "LIVE" if filterLiveAccounts else "TEST" }}</strong>
     {% else %}
        Transactions
     {% endif %}
   </h1>
+
+  {% if allServiceTransactions %}
+    {% if filterLiveAccounts %}
+      <p><a class="govuk-link govuk-link--no-visited-state" href="{{ routes.formattedPathFor(routes.allServiceTransactions.indexStatusFilter, 'test') }}">Switch to test</a></p>
+    {% else %}
+      {% if hasLiveAccounts %}
+        <p><a class="govuk-link govuk-link--no-visited-state" href="{{ routes.allServiceTransactions.index }}">Switch to live</a></p>
```

Comment:
This will be available immediately (without setting ENABLE_TEST_REPORTS). There will be small window that users see `switch to test` and directed to old instance. Fine as it is too as it is very unlikely lot of users see this or access test transactions

Reply:
Yeah it seemed like the only users that would see this are already familiar with this tool so the fact it works for the test environment too shouldn't throw them, a good point though and should be fairly easy to change if we need to